config/job.js

```diff
@@ -72,6 +72,7 @@ const SCHEMA_REQUIRES = Joi.alternatives().try(
     Joi.array().items(SCHEMA_REQUIRES_VALUE),
     SCHEMA_REQUIRES_VALUE
 );
+const SCHEMA_BLOCKEDBY = Joi.array().items(SCHEMA_JOBNAME);
```

Comment:
Wondering if we will support a single item or an array; also can blockedBy values be external triggers?